### PR TITLE
Get user from Host for ssh dir

### PR DESCRIPTION
**Pull request by myakove**

### CODE REVIEW

File: rrmngmnt/host.py [hunk 1]
```diff
@@ -112,12 +112,20 @@ def executor(self, user=None, pkey=False):
             user = copy.copy(self.root_user)
         return ssh.RemoteExecutor(user, self.ip, use_pkey=pkey)
 
-    def run_command(self, command):
+    def run_command(
+        self, command, input_=None, tcp_timeout=None, io_timeout=None
```
Comment (cynepco3hahue) [inline rrmngmnt/host.py]: what point to add input_=None, tcp_timeout=None, io_timeout=None if not pass it to self.executor().run_cmd

File: rrmngmnt/ssh.py [hunk 2]
```diff
@@ -56,7 +55,7 @@ def __init__(self, executor, timeout=None, use_pkey=False):
             self._ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
             if use_pkey:
                 self.pkey = paramiko.RSAKey.from_private_key_file(
-                    ID_RSA_PRV
+                    ID_RSA_PRV % self._h.user.name
```
Comment (nellyc) [inline rrmngmnt/ssh.py]: os.path.expanduser

and I still cant find _h

Comment (myakove) [inline rrmngmnt/ssh.py]: I think _h is in Host

Comment (myakove) [inline rrmngmnt/ssh.py]: rrmngmnt/ssh.py:            self._h = host

**Pull request merged**